pdns/recursordist/settings/table.py

```diff
@@ -2998,6 +2998,7 @@
    udp_source_port_avoid:
    - 5300
    - 11211
+   - 4791
```

Comment:
Consider keeping this list sorted by port number.

Reply:
i done this